testplan/testing/multitest/base.py

```diff
@@ -1154,3 +1161,24 @@ def _split_by_exec_group(testcases):
             serial_cases.append(testcase)
 
     return serial_cases, parallel_cases
+
+
+def _add_runtime_info(param_report):
+    """
+    Add runtime information to parametrized group report.
+    :param param_report: parametrized group report
+    :return: the parametrized group report with its runtime information
+    """
+    group_start_time = param_report.entries[0].timer["run"].start
+    group_end_time = param_report.entries[0].timer["run"].end
+    for testcase in param_report.entries:
+        timer = testcase.timer
+        start_time = timer["run"].start
+        end_time = timer["run"].end
+        if start_time < group_start_time:
```

Comment:
how about something like this?

```suggestion
        group_start_time =  start_time if group_start_time is None else min(group_start_time, start_time)
```